website/blog/2021-06-13-Hydra_1.1.md

```diff
@@ -0,0 +1,129 @@
+- Support to parameter conversion strategy was added via the `_convert_` config key.
+
+Learn more [here](/docs/next/advanced/instantiate_objects/overview).
+
+### Hydra callbacks
+A new experimental mechanism for user defined callbacks was added.
+Callbacks enables user code to be executed automatically at various points in the lifecycle of your application. 
+There are many potential use cases for this, for example automatic registration with your 
+favorite experiment tracking service.
+
+Learn more [here](/docs/next/experimental/callbacks).
+
+### OmegaConf 2.1 highlights
+OmegaConf 2.1 includes many enhancements, bug fixes, and performance improvements.
+
+#### Relative interpolations
+Relative interpolations enable accessing a config node relative to the node defining the interpolation:
+```yaml
+x: 10
+b:
+  y: 20
+  a: {x}    # 10, absolute interpolation
+  d: ${.y}  # 20, relative interpolation
+  e: ${..x} # 10, relative interpolation
+```
+
+#### Nested interpolations
+OmegaConf 2.1 adds a new interpolation grammar supporting more sophisticated usage of interpolations.  
+In the following example, the default value to use if the environment variable `DB_USER` does not exist is defines in the `default_user` config node:  
+```yaml
+default_user: root
+db_user: ${oc.env:DB_USER,${default_user}}
+```
+
+#### More powerful custom resolvers
+OmegaConf custom resolvers can now access parent config node or the config root by defining
+keyword parameters named `_parent_` and `_root_`.
+
+In the example below, we use `_parent_` to implement a sum function that defaults to 0 if the node does not exist:
+
+```python
+def sum(a, b, *, _parent_):
+  return _parent_.get(a, 0) + _parent_.get(b, 0)
+
+OmegaConf.register_new_resolver("sum", sum, use_cache=False)
+cfg = OmegaConf.create({
+  "node": {
+    "a": 1,
+    "b": 2,
+    "a_plus_b": "${sum:a,b}",
+    "a_plus_z": "${sum:a,z}",
+  },
+})
+print(cfg.node.a_plus_b)  # 3
+print(cfg.node.a_plus_z)  # 1
+```
+
+### Other notable improvements
+- Config composition, especially for large configs - is significantly faster. 
+- `OmegaConf.resolve(cfg)` can be used for in-place interpolation resolution on a config object
+- Improved compatibility of OmegaConf config objects with plain dict and list.
+- Support for bracketed style (`foo.bar` is equivalent to `foo[bar]`), this covers interpolations and `OmegaConf.{update, select}` usage.
+- PyDev.Debugger integration for easier debugging of config objects in PyCharm and VSCode
+
+### Migrating from 1.0
+Hydra 1.1 is a major release. For most people, migrating from 1.0 to 1.1 will be smooth.
+However, there are some breaking changes listed in the release notes of OmegaConf 2.1 and Hydra 1.1.
+Most changes comes with a deprecation warning pointing to a specific migration guide page.
+Please feel free to reach out for help if you see a change in behavior that is not mentioned in the release notes.
```

Comment:
Maybe clarify how to "reach out for help"? (is there a place in the doc we can point out to that lists the various ways to ask for help?)

Reply:
Good idea.